### ISSUE TITLE
Responsive extra with Turbolinks

### ISSUE THREAD

**Issue by enzinia** [ddnexus/pagy#64]
I constantly have a repeated error in the console when resizing browser window:
```
Uncaught TypeError: Cannot read property 'forEach' of undefined
    at render (pagy.self-8fb09da8735807829df91dbb760bbd342dd9de76a6e68fa5d7da948200b00ea8.js?body=1:56)
```
Rails 5.2, Turbolinks 5.1.1

How to repeat:
1. Use pagy responsive extra
2. Call Pagy.init() in `turbolinks:load` event listener
3. Switch page (go to link via Turbolinks)

The more we change the pages the faster the errors grow. Possibly related to binding events to `window`

![responsive_with_turbolinks](https://user-images.githubusercontent.com/26239269/42386434-7034a2dc-817a-11e8-8727-988f82ee9c6d.png)

**Comment by ddnexus** [comment]
Yeah, I got that too a few times, but when I check it again the `series[width]` is not undefined anymore. Thanks for the step list, I will try to look into it soon

**Comment by ddnexus** [comment]
@enzinia I tried it in a few browsers and I couldn't reproduce it. Please, could you share an app, or even just a HTML file with the javascript? I have no clue about why and when this happens.

**Comment by ddnexus** [comment]
@enzinia
I still have to figure out how to refactor it to avoid the problem, but I found the cause.  I don't need any app nor HTML file anymore. Thanks.

**Comment by enzinia** [comment]
Good to hear!

I'll also try to find way to fix it.

**Comment by ddnexus** [comment]
@enzinia thanks by please, don't waste your time on this one since it will probably not be just an isolated fix but a refactoring of a few things that have to work together

**Comment by enzinia** [comment]
Roger.

**Comment by alec-implicitdesign** [comment]
This bug seems to be back on the current master branch.  It seems like the particular fix that was mentioned here was removed again in the meantime.
